### PR TITLE
Routing, API calls (axios), vuex, Login & Dashboard pages.

### PR DESCRIPTION
**Pull request by michalboruch**
### Routing, we've got 3 pages:
- Home `/`
- Login page `/login`
- Dashboard (home for logged users) `/dashboard`

Redirections:
- from login page to dashboard, after successful login
- to login page after logout

###  Page header 
Common nav and login/logout buttons

### API calls
Added axios to handle API calls

### VUEX
Basic config.
Keeping state `is_auth`

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+
 export default {
-  name: 'App'
-}
+  name: 'App',
+  setup () {
+    store.commit('initStore')
+  },
+  components: {
+    'app-header': Header
```
Comment (janeek1995) [inline src/App.vue]: We need to establish name convention. I use for example just name of the component.

File: src/components/Header.vue [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  <header>
+    <div class="logo">
+      <router-link to="/">Minusz labs</router-link>
+    </div>
+    <nav>
+      <ul>
+        <li v-if="!is_auth">
+          <router-link to="/login">Log In</router-link>
+        </li>
+        <li v-if="is_auth">
```
Comment (janeek1995) [inline src/components/Header.vue:8]: I think that you could use here else

File: src/components/Header.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<template>
+  <header>
+    <div class="logo">
+      <router-link to="/">Minusz labs</router-link>
```
Comment (janeek1995) [inline src/components/Header.vue:4]: What do you think about directing by name instead of path ?

File: src/App.vue (hunk 1 above)
Comment (michalboruch) [reply src/App.vue]: done

File: src/components/Header.vue (hunk 3 above)
Comment (michalboruch) [reply src/components/Header.vue:4]: done

**Review by janeek1995 — APPROVED** [review]

**Pull request merged**